### PR TITLE
Position: Simplify fraction support test

### PR DESCRIPTION
**Pull request by scottgonzalez**
Also makes the test lazy to avoid any potential layouts/recalculations during
initialization.

Fixes #9898
Ref #9899

### CODE REVIEW

File: tests/unit/position/position_core.js [hunk 1]
```diff
@@ -763,4 +761,6 @@ test( "bug #8710: flip if flipped position fits more", function() {
 	}, "no flip - top fits less" );
 });
 
+TestHelpers.testJshint( "position" );
```
Comment (scottgonzalez) [inline tests/unit/position/position_core.js]: This is whacky, but having this run first causes intermittent failures in IE.

Comment (jzaefferer) [inline tests/unit/position/position_core.js]: I still wonder if this could be fixed by `positon: absolute` on the support test element.

Comment (scottgonzalez) [inline tests/unit/position/position_core.js]: Indeed it does :-)

**Comment by jzaefferer** [comment]
:+1:

**Comment by scottgonzalez** [comment]
Landed in 3970e8c68146a3645f09d54735a40d5cf7eeef9e.

**Pull request closed without merging**